BloodManagmentSystem/Persistance/EntityConfigurations/DonorConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BloodManagmentSystem.Core.Models;
+using System.Data.Entity.ModelConfiguration;
+
+namespace BloodManagmentSystem.Persistance.EntityConfigurations
+{
+    public class DonorConfiguration : EntityTypeConfiguration<Donor>
+    {
+        public DonorConfiguration()
+        {
+            Property(d => d.Name)
+                .IsRequired()
+                .HasMaxLength(255);
```

Comment:
Long name can cause displaying problems, max of 30 chars should be enough

Reply:
Someone could enter surname beside first name, so maybe 50 chars?